enterprise/backend/test/metabase_enterprise/serialization/test_util.clj

```diff
@@ -72,7 +72,7 @@
         data-source       (mdb.data-source/raw-connection-string->DataSource connection-string)]
     ;; DB should stay open as long as `conn` is held open.
     (with-open [_conn (.getConnection data-source)]
-      (with-db data-source (mdb/setup-db!))
+      (with-db data-source (mdb/setup-db! :create-sample-content? false)) ; skip sample content for speedy tests. this doesn't reflect production
```

Comment:
> I feel that this comment belongs in every test

I'm not sure I understand. I have added this comment in every case where `:create-sample-content?=false` in tests for this reason. maybe you mean something more than that?

Yeah good idea about adding it to the docstring, I will do that

Reply:
This comment came from other tests disabling the sample content without any comment, e.g. the e2e tests. Since this was the only test with any comment until much further down in the diff, I thought it might be the only spot. I think it's worth noting the relevant reason for every test that explicitly disables it.

Reply:
Basically a repeat of my later comment https://github.com/metabase/metabase/pull/40753#discussion_r1551276288

Reply:
what tests are missing the comment? I thought I got everything

Reply:
resolving this because we switched to other comment thread. On the docstring, I have added it in 8dc50dd90ef9683979ed4d047d88992c8720c826